scripts/install.sh

```diff
@@ -73,8 +73,16 @@ configure_systemd() {
         $SUDO useradd -r -s /bin/false -m -d /usr/share/ollama ollama
     fi
 
-    status "Creating ollama systemd service..."
-    cat <<EOF | $SUDO tee /etc/systemd/system/ollama.service >/dev/null
+    status "Adding current user to ollama group..."
+    $SUDO usermod -a -G ollama $(whoami)
+    # Allow the ollama user to access the current user's files
+    # TODO: This can be removed when model creation does not require access to the current user's files
+    status "Adding ollama to the current user group..."
+    $SUDO usermod -a -G $(whoami) ollama
+
+    if [ ! -f /etc/systemd/system/ollama.service ]; then
```

Comment:
This means the install script will not update the service file of an existing install. It's not ideal since there may be changes in the service file itself.